server/src/concepts/user/useCases/createNewUser.js

```diff
@@ -0,0 +1,32 @@
+// import { createUser } from "../repositories/commands.js";
+import User from "../model/User.js";
+import bcrypt from "bcrypt";
+
+const verifyIfUserExists = async (email) => {
+  const isEmailExist = await User.findOne({ email: email });
```

Comment:
It would be better to create simple function to find user with email in repositories/queries and just use it here